python3-get-framework-k8s-api-version.spec

```diff
@@ -17,16 +17,26 @@
 
 
 %define upstream_name get-framework-k8s-api-version
+# this code is intended to be run in an instance of any of the supported CSPs,
+# where this info is available.
+%define is_azure "%(sudo dmidecode -s bios-vendor | grep Microsoft)"
+%define is_google "%(sudo dmidecode -s bios-vendor | grep Google)"
 
 Name:           python3-get-framework-k8s-api-version
-Version:        0.0.1
+Version:        0.0.3
 Release:        0
 Summary:        latest version of k8s API server
 License:        GPL-3.0+
 Url:            https://github.com/SUSE-Enceladus/get-framework-k8s-api-version
 Source0:        %{upstream_name}-%{version}.tar.bz2
 Requires:       python3
 Requires:       python3-requests
+%if %{is_azure}
+Requires: python3-azure-mgmt-containerservice
+Requires: python3-msrestazure
+Requires: python3-azure-common
+Requires: python3-azure-identity
+%endif
```

Comment:
There are major changes happening in the Azure SDK w.r.t. authentication and these are incompatible changes. As such the package versions need to have limits set i.e. `>` or less then depending on which path the used authentication follows "old" vs. "new"